### ISSUE TITLE
Double check with run-time

### ISSUE THREAD

**Issue by Yijunmaverick** [Ben-Louis/Deep-Image-Analogy-PyTorch#2]
Hi, thanks for this great pytorch implementation. I run the code for the demo image pair on GPU and it takes about 40min. The result is good. Just double check, is my run-time close to yours?

**Comment by Ben-Louis** [comment]
Yes. I ran the code on VM of GCP which has a 4-core CPU and a K80, and it takes around 30 mins. The patchmatch process on low level features (large size) is quite slow because it does not run on GPU

**Comment by harveyslash** [comment]
@Ben-Louis Thanks for giving credit for my implementation. 
I would like to let you know that I have also released a GPU version of PatchMatch , written in 
https://github.com/harveyslash/PatchMatch

**Comment by tisawe** [comment]
@harveyslash Do you have a GPU version of PatchMatch which works with your PyTorch version of Deep Image Analogy?

**Comment by harveyslash** [comment]
Yes. It is a drop in replacement for the cpu version

**Comment by qhanson** [comment]
Have you checked the PatchMatch both in gpu version and the cpu version?? I use the same image as the parameters, however, the constructed image is a noise.

**Comment by qhanson** [comment]
@harveyslash  Thanks for your implementation for the GPU version. Remind that anyone must normalize your own images before using the PatchMatch algorithm.

However, using the GPU version PatchMatch, the result is a little different from the CPU version. I am still debugging on it.

**Comment by qhanson** [comment]
@Ben-Louis  When I run examples, most of the run-time is spent on the deconvolution process... I don't think it is the problem of PatchMatch process.

**Comment by vikasTmz** [comment]
updates?
